### DISCUSSION TITLE
CCTV-16 [CN]

### DISCUSSION THREAD

**Discussion by rustemzhang** [iptv-org/community#870] [Show and tell]
### Channel ID

CCTV16.cn

### Stream URL

http://ott.mobaibox.com/PLTV/3/224/3221228127/index.m3u8

### Notes

#EXTINF:-1 tvg-id="CCTV16.cn", CCTV-16 (4k)
http://ott.mobaibox.com/PLTV/3/224/3221228127/index.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Does it only answers if the IP is said to be Chinese, or does it locks VPNs of some sort ? On my side, it does not seem to answer.

**Comment by freearhey** [comment]
http://ott.mobaibox.com/PLTV/3/224/3221228127/index.m3u8 (offline)
<img width="438" alt="image" src="https://user-images.githubusercontent.com/7253922/232257979-0eba5282-38d7-4cbc-8aa3-29e9baa868f8.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/cn.m3u#L58-L59
